engine/pom.xml

```diff
@@ -23,7 +23,7 @@
     <distributionManagement>
         <site>
             <id>site</id>
-            <url>http://validator.hibernate.org</url>
+            <url>http://hibernate.org/validator</url>
```

Comment:
+1 for http://hibernate.org/validator, it's nicer to the user to avoid the re-direct. But guys, let's spend our cycles on more valuable stuff ;)